Cargo.toml

```
@@ -14,10 +14,10 @@ keywords = ["tree-sitter", "traverse", "traversal", "tree"]
 categories = ["algorithms", "data-structures", "no-std"]
 
 [dependencies]
-tree-sitter = {version=">= 0.19, < 0.21", optional=true}
+tree-sitter = {version="0.24", optional=true}
```

@citronneur  do you mind updating your `tree-sitter-traversal2` crate so that it supports `tree-sitter` 0.25 while we wait for the maintainer?
```suggestion
tree-sitter = {version=">=0.24", optional=true}
```